cmd/launcher/desktop.go

```diff
@@ -186,6 +193,22 @@ func monitorParentProcess(logger log.Logger) {
 	}
 }
 
+func parentProcessId() (int, error) {
+	if runtime.GOOS != "darwin" {
+		return os.Getppid(), nil
+	}
+
+	// On macOS, we run desktop as `launchctl asuser <UID> sudo --preserve-env -u <username> launcher desktop`, which
+	// means the parent process of the current process is the sudo process. We need to monitor one level above that
+	// to be monitoring the launcher root process.
+	sudoProcess, err := ps.FindProcess(os.Getppid())
```

Comment:
could use `process.NewProcess()` (from `github.com/shirou/gopsutil/process` already imported) rather than importing an additional pkg

Reply:
Oh nice, thank you!

Reply:
Oh, interesting -- discovered when testing the new logging that gopsutil/process isn't able to find the parent process whereas go-ps could -- looks like the two packages use different methods to find the parent. In light of that, I'm removing this method to rely just on the PPID command-line flag.